src/Mod/Draft/draftguitools/gui_groups.py

```diff
@@ -68,6 +71,7 @@ def GetResources(self):
                     "Create a group first to use this tool.")
 
         d = {'Pixmap': 'Draft_AddToGroup',
+             'Accel':"M",
```

Comment:
I think Yorik mentioned in the past that we don't want to have single letter shortcuts because they easily collide with other shortcuts that are previously defined.

Reply:
come on it's a Blender shortcut to move slected object

Reply:
There are other, existing FreeCAD shortcuts that begin with "M" (Draft Move is M,V for example). If you make one with only "M", you are blocking all others to work properly as you are swallowing their first letter. You can use M for yourself if you like it, but it cannot be merged like this as a default for everybody because it harms existing functionality.

Reply:
```suggestion
             'Accel':"M,G",
```